gtk4-macros/src/lib.rs

```diff
@@ -15,6 +15,51 @@ use proc_macro::TokenStream;
 use proc_macro_error::proc_macro_error;
 use syn::{parse_macro_input, DeriveInput};
 
+/// That macro includes and compiles blueprint file by path relative to project rood
+///
+/// It expected to run inside composite_template_derive, not by users
+#[cfg(feature = "blueprint")]
+#[proc_macro]
+#[proc_macro_error]
+pub fn include_blueprint(input: TokenStream) -> TokenStream {
```

Comment:
```suggestion
#[doc(hidden)]
pub fn include_blueprint(input: TokenStream) -> TokenStream {
```
To avoid people finding about it and misusing it